### DISCUSSION TITLE
What is the best release/environment for a lightweight setup?

### DISCUSSION THREAD

**Discussion by brainlessbitch** [dnschneid/crouton#4947] [Q&A]
Title is pretty self-explanatory, I wanna minimize the amount of CPU and RAM used by the chroot. I'd love some help, Thank you ^w^

**Comment by CroutonIsFun** [comment]
There is no absolute answer, but a lot of people like xfce. The default release xenial is out of date, so I recommend focal, even though the installer says it's unsupported. All new releases except for Debian are unsupported.

lxde is very lightweight, but always throws an error when booting into the desktop. Once you click OK, the error goes away, and everything just works. lxde requires some tweaking to get the touchpad working and adjust the double click speed.

To install with crouton, you will need to revert to 110 or use the silence branch. See the earlier discussions.

**Reply by satoshiHellomoto** [reply]
![screen](https://github.com/dnschneid/crouton/assets/125427783/fdd2d401-67b7-4d0e-87ff-a003bf3caab8)

I recommend trying out different virtual environments tax the machine. I personally bought mine very short while ago, and played with it for a month. there are ways to flag certain hardware in other operating systems, for example - flagging all of my CPU threads in BSD versus the half that came shut off. Usually the lower level .sh , scripts , ... are going to be set and re-booted into from crosh, and specifically chronos log in, where Ctrl alt refresh/forward/back arrows formerly (F)function keys flip the user into a prompt to log in as chronos. There you can set how the machine uses hardware. RAM is usually 1501 MB - 1802 MB per virtual instantiation out of the set 2774 MB.

**Comment by ghost** [comment]
Typically XFCE or LXDE will have the least impact on the system.
